rkt/run.go

```diff
@@ -445,7 +445,9 @@ func (e *envFileMap) Set(s string) error {
 
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
-		// TODO parse '\' to skip new line character
+		if len(scanner.Text()) == 0 {
+			continue
```

Comment:
This sounds ok.

But I just noticed that the skipLine() test with `len(s) == 0` below for "Empty line" is wrong because `strings.SplitN(..., "=", 2)` should never return an empty array of string.